src/app/campaign/flight/flight-form.component.ts

```diff
@@ -26,7 +26,6 @@ export class FlightFormComponent implements OnInit {
   @Output() flightDeleteToggle = new EventEmitter(true);
 
   flightForm: FormGroup;
-  matcher = new FlightFormErrorStateMatcher();
```

Comment:
this wasn't used anymore since we changed datepickers to update on blur, removed